--- a/package/qt5/qt5webengine/Config.in        2021-12-15 23:03:51.440000000 +0100
+++ b/package/qt5/qt5webengine/Config.in        2021-12-15 23:02:54.870000000 +0100
@@ -47,7 +47,6 @@
        select BR2_PACKAGE_WEBP_DEMUX
        select BR2_PACKAGE_QT5BASE_DBUS
        select BR2_PACKAGE_QT5BASE_EGLFS
-       select BR2_PACKAGE_QT5BASE_FONTCONFIG
        select BR2_PACKAGE_QT5BASE_ICU
        select BR2_PACKAGE_QT5BASE_GUI
        select BR2_PACKAGE_QT5BASE_WIDGETS
