diff --git a/package/qt5/qt5webengine/Config.in b/package/qt5/qt5webengine/Config.in
index 4e923c7f31..d5ad2d13ef 100644
--- a/package/qt5/qt5webengine/Config.in
+++ b/package/qt5/qt5webengine/Config.in
@@ -47,7 +47,6 @@ config BR2_PACKAGE_QT5WEBENGINE
        select BR2_PACKAGE_WEBP_DEMUX
        select BR2_PACKAGE_QT5BASE_DBUS
        select BR2_PACKAGE_QT5BASE_EGLFS
-       select BR2_PACKAGE_QT5BASE_FONTCONFIG
        select BR2_PACKAGE_QT5BASE_ICU
        select BR2_PACKAGE_QT5BASE_GUI
        select BR2_PACKAGE_QT5BASE_WIDGETS

